### ISSUE TITLE
cannot find module gatsby-transformer-inline/package.json

### ISSUE THREAD

**Issue by Xstoudi** [konstantinmuenster/gatsby-theme-portfolio-minimal#39]
I cannot build the template.

Node version : 18.14.0
NPM version : 9.5.0

```
λ npm run build -- --verbose                                                                             
                                                                                                         
> REDACTED@1.0.0 build                                                                                 
> gatsby build --verbose                                                                                 
                                                                                                         
verbose set gatsby_log_level: "verbose"                                                                  
verbose set gatsby_executing_command: "build"                                                            
verbose loading local command from: REDACTED\node_modules\gatsby\dist\commands\build.js    
verbose running command: build                                                                           
verbose Running build in "production" environment                                                        
success compile gatsby files - 2.146s                                                                    
success load gatsby config - 2.784s                                                                      
                                                                                                         
 ERROR  UNKNOWN                                                                                          
                                                                                                         
plugin "gatsby-transformer-inline-svg threw the following error:                                         
 Cannot find module 'gatsby-transformer-inline-svg/package.json'                                         
```

The dependencies are the following:
```
  "dependencies": {
    "gatsby": "^5.9.0",
    "gatsby-theme-portfolio-minimal": "^4.5.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
```

**Comment by konstantinmuenster** [comment]
Hey @Xstoudi! Thanks for opening this issue. That's indeed weird since the package is included as a theme dependency. I need to look into it.

If you want to have a quick fix meanwhile, you can add the plugin manually to your deps:

```
  "dependencies": {
    "gatsby": "^5.9.0",
    "gatsby-theme-portfolio-minimal": "^4.5.0",
    "gatsby-transformer-inline-svg": "^1.2.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
```

**Comment by Xstoudi** [comment]
It works for now =)

**Comment by konstantinmuenster** [comment]
@Xstoudi Just released a new version ([v4.5.1](https://github.com/konstantinmuenster/gatsby-theme-portfolio-minimal/releases/tag/v4.5.1)) that should fix this :)
